clippy_lints/src/mutex_atomic.rs

```diff
@@ -48,14 +51,23 @@ declare_clippy_lint! {
     /// Checks for usage of `Mutex<X>` where `X` is an integral
     /// type.
     ///
-    /// ### Why is this bad?
+    /// ### Why is restrict bad?
```

Comment:
Sorry, is this a typo?

Reply:
See https://github.com/rust-lang/rust-clippy/pull/12838.

Reply:
So it should "Why restrict this?", not "Why is restrict bad?", right?

Reply:
Oops sorry, it was a copy paste error. Fixed.